modules/ROOT/pages/Development/Cpp/setup.adoc

```diff
@@ -74,102 +74,11 @@ to assist with this.
 
 Within this new file you will need to add the following configuration text.
 
-Make sure to replace all instances of `YourModReference` with your actual mod reference.
+Make sure to replace all instances of `PLUGIN_NAME` with your actual mod reference.
 
 The latest version of this template can be found in the
 https://github.com/satisfactorymodding/SatisfactoryModLoader/blob/master/Mods/Alpakit/Templates/CPPAndBlueprintBlank/Source/PLUGIN_NAME/PLUGIN_NAME.Build.cs[Starter Project Repository as an Alpakit template].
 
```

Comment:
Could you reorder+rephrase this so it's saying to visit the template file, then telling you how to edit it? Right now it's in reverse. I'm on mobile so cant use the Suggest feature easily

Reply:
I swapped the paragraphs' order, I think it's reasonable enough but I'd like to know what your thoughts are.